### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Alexr16**
* Added JavaScript + HTML + CSS linters.
*  Mobile Menu created.
* script.js file created.
* Action, mobile menu appears, created.
* Action, mobile menu disappears, created.
* Action, click any of the mobile menu options, the mobile menu disappears, created.
* Action, click  of the mobile menu options, a correct part of the page displayed, created.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const navMenu = document.querySelector('.list');
+const icons = document.querySelectorAll('.menu-icon');
+const hamburguerIcon = document.querySelector('#hamburguer');
+
+hamburguer.addEventListener('click', () => {
+  navMenu.classList.toggle('active');
+  icons.forEach((icon) => {
+    icon.classList.toggle('close-menu');
+  });
+});
```
Comment (ReshyResh) [inline script.js]: - [ ] Please make sure that the menu element stays intact even after scrolling down. Currently, as the user scrolls down with the menu active, the top section of the menu stays in the header, leaving a small empty space at the top and losing the icon to close the menu. You could add a condition that adds the property `overflow: hidden` to the body when the menu is active or fix the menu itself, please make sure to address this issue.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi team! 

### Good job so far!
  - ✔ Design matches the Figma template.
  - ✔ Good UI with pleasant animations.
  - ✔ Correctly setup ESLint with no errors.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alexr16** [comment]
* Fixed Small empty space at the top to the menu. 
* Added condition that adds the property overflow: hidden to the body .

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi Team, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**